cmd/utils.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/redhat-developer/odo/pkg/component"
+	"github.com/redhat-developer/odo/pkg/occlient"
+)
+
+// printDeleteAppInfo will print things which will be deleted
+func printDeleteAppInfo(client *occlient.Client, appName string, currentProject string) error {
+	componentList, err := component.List(client, appName, currentProject)
+	if err != nil {
+		return errors.Wrap(err, "failed to get Component list")
+	}
+
+	for _, cmpnt := range componentList {
+		_, _, componentURL, appStore, err := component.GetComponentDesc(client, cmpnt.Name, appName, currentProject)
+		if err != nil {
+			return errors.Wrap(err, "unable to get component description")
+		}
+		fmt.Println("Component", cmpnt.Name, "will be deleted:")
```

Comment:
The ":" looks strange when the component doesn't have any URLs or storages.

```
▶ odo app delete app
Component backend will be deleted:
Component frontend will be deleted:
This component is externally exposed, and the URL will be removed
Are you sure you want to delete the application: app? [y/N]
```

This looks better:
```
▶ odo app delete app
Component backend will be deleted.
Component frontend will be deleted.
  This component is externally exposed, and the URL will be removed
Are you sure you want to delete the application: app? [y/N]
```